software/runtime-mbed.md

```diff
@@ -20,7 +20,7 @@ review-with: davidw, joef
 
 This page provides an overview of the micro:bit runtime - software that runs on a micro:bit to support the majority of the micro:bit programming languages. It can help you understand how the micro:bit works, and also will help you understand where to start if you want to dive deeper into the micro:bit, write support software for your own micro:bit hardware extensions, and tailor or improve something on the micro:bit.
 
-![Architectural Diagram](/docs/software/assets/dal-4032c.png)
+![Architectural Diagram](/docs/software/assets/software-overview.png)
```

Comment:
Does this render correctly? The new image path is `software/assets/software-overview.png`, is `/docs/` a directory where all this is built inside? So the final URL would be http://tech.microbit.org/docs/software/assets/software-overview.png, instead of http://tech.microbit.org/software/assets/software-overview.png?

Reply:
Yes, all this is pulled into a folder called '/docs/' so this image will replace https://tech.microbit.org/docs/software/assets/dal-4032c.png

Reply:
Great, thanks for clarifying!